README.md

```diff
@@ -1,11 +1,25 @@
 # RAPIDS [devcontainers](https://containers.dev/)
 
-This repository contains features and workflows for building development containers to support local dev and CI for NVIDIA [RAPIDS](https://github.com/rapidsai), [CCCL](https://github.com/nvidia/cccl), and [Legate](https://github.com/nv-legate).
+This repository contains features and workflows for building development
+containers to support local dev and CI for NVIDIA
+[RAPIDS](https://github.com/rapidsai), [CCCL](https://github.com/nvidia/cccl),
+and [Legate](https://github.com/nv-legate).
+[Devcontainers](https://containers.dev/) are an open standard for specifying the
+creation and execution of Docker containers for developing a codebase. It's like
+using a docker image to develop, but there's some extra configuration and
+installation that can be done. It also provides some alternative ways of
+composing functionality and configuration that can augment Docker's
+capabilities.
```

Comment:
Removing line breaks in paragraphs.

```suggestion
This repository contains features and workflows for building development containers to support local dev and CI for NVIDIA [RAPIDS](https://github.com/rapidsai), [CCCL](https://github.com/nvidia/cccl), and [Legate](https://github.com/nv-legate).

[Devcontainers](https://containers.dev/) are an open standard for specifying the creation and execution of Docker containers for developing a codebase. It's like using a docker image to develop, but there's some extra configuration and installation that can be done. It also provides some alternative ways of composing functionality and configuration that can augment Docker's capabilities.
```

Reply:
We can do this if you prefer, but the line breaks do not appear in rendered markdown. The reason for the linebreaks is that it makes navigating the raw .md file easier.

Reply:
I don't have a strong opinion (this is a personal-preference style thing), but will just add that I tend to use line breaks like this because they also make git diffs and the git blame a bit more informative and easier to use.

`*backs away from style debate*`